Changed skillcrush to denise and also put in two paragraphs at the bo…

Denisealisen opened the pull request:
…ttom of the others

twosavoie commented:
Hi @Denisealisen,

This is a great start! Just a couple of details to take care of. 

We would like your advice file to be in the advice folder. Once the advice file is in the advice folder, add a link to it in the index.html file. You can test to see if the link is working by opening the project in your browser and clicking on it. Do you see your advice?

Once you've made those changes, in the command line run `git add -A` (the capital A is important), commit, then push your advice branch to GitHub. Your changes will join your pull request here.

Let us know if you need any help!

Best,
Lisa

Denisealisen commented:
I pushed through the updated one on the advice branch.

twosavoie commented:
Hi @Denisealisen,

This looks so much better!! Just a couple of small details to take care of. You can have a look at the differential [here](https://github.com/skillcrush/git-practice/pull/1281/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L5).

The link to the main.css file in the <head> of the index.html file has been changed to remove the folder from the file path. Please return it to `<link rel="stylesheet" type="text/css" href="css/main.css">`. Then in your advice file, you'll want to update the file path to the css file - adding the folder name to the front.

Another student's link was deleted from the index.html file: `<a href="advice/danielle-rojas-advice.html">Danielle R's Advice</a><br>`. Can you add it back and then add and commit these changes and push your advice branch to GitHub? It will update your pull request and we'll take a look!

Let us know if you have any questions!

Best,
Lisa

Denisealisen commented:
Thank you, I will do that.
What do you mean by adding the folder name to the front?

On Sat, Jun 1, 2024 at 12:15 PM Lisa Savoie ***@***.***>
wrote:

> Hi @Denisealisen <https://github.com/Denisealisen>,
>
> This looks so much better!! Just a couple of small details to take care
> of. You can have a look at the differential here
> <https://github.com/skillcrush/git-practice/pull/1281/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L5>
> .
>
> The link to the main.css file in the of the index.html file has been
> changed to remove the folder from the file path. Please return it to <link
> rel="stylesheet" type="text/css" href="css/main.css">. Then in your
> advice file, you'll want to update the file path to the css file - adding
> the folder name to the front.
>
> Another student's link was deleted from the index.html file: <a
> href="advice/danielle-rojas-advice.html">Danielle R's Advice</a><br>. Can
> you add it back and then add and commit these changes and push your advice
> branch to GitHub? It will update your pull request and we'll take a look!
>
> Let us know if you have any questions!
>
> Best,
> Lisa
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/git-practice/pull/1281#issuecomment-2143518974>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGVBDB6DZE52BZ2TAUY6GDLZFH6UVAVCNFSM6AAAAABIOA7XPWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCNBTGUYTQOJXGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

twosavoie commented:
Sounds great @Denisealisen,

This is an awesome question. File paths are so important to get comfortable with as a web developer. Since the css files are in the css folder, the file path needs to include a reference to the folder. 

The file path in the index.html file can be returned to what it was in the starter code. Because the index.html file and the css folder are at the same level, you can add `css` to the front of the file path to complete the link. Have a look at the diff linked in the last message to see the before and after.

Because your advice file is in an advice folder, the path to your css file is a little bit different. It needs to go up a level (to the root of the project folder), then to the css folder to find your css file. When we specify that we want to go up a level, we add `../` to the front of the file path. Then we add the folder we have to go into to find the css file. The path would look like this: `"../css/denise-wilson-advice.css"`

Does that help? Reach out if there's anything I can clarify! Keep going - you've got this!  

Best,
Lisa

Denisealisen commented:
Thank you for the clarification.

On Sun, Jun 2, 2024 at 12:08 AM Lisa Savoie ***@***.***>
wrote:

> Sounds great @Denisealisen <https://github.com/Denisealisen>,
>
> This is an awesome question. File paths are so important to get
> comfortable with as a web developer. Since the css files are in the css
> folder, the file path needs to include a reference to the folder.
>
> The file path in the index.html file can be returned to what it was in the
> starter code. Because the index.html file and the css folder are at the
> same level, you can add css to the front of the file path to complete the
> link. Have a look at the diff linked in the last message to see the before
> and after.
>
> Because your advice file is in an advice folder, the path to your css file
> is a little bit different. It needs to go up a level (to the root of the
> project folder), then to the css folder to find your css file. When we
> specify that we want to go up a level, we add ../ to the front of the
> file path. Then we add the folder we have to go into to find the css file.
> The path would look like this: "../css/denise-wilson-advice.css"
>
> Does that help? Reach out if there's anything I can clarify! Keep going -
> you've got this!
>
> Best,
> Lisa
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/git-practice/pull/1281#issuecomment-2143699799>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGVBDB4RWDZ6DGY2ASMZYTLZFKSDNAVCNFSM6AAAAABIOA7XPWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCNBTGY4TSNZZHE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Denisealisen commented:
I fixed the css main and added Danielle back in and add the changes to my
css file as well and added the commits I pushed it through.

On Mon, Jun 3, 2024 at 6:13 AM Denise Wilson ***@***.***>
wrote:

> Thank you for the clarification.
>
> On Sun, Jun 2, 2024 at 12:08 AM Lisa Savoie ***@***.***>
> wrote:
>
>> Sounds great @Denisealisen <https://github.com/Denisealisen>,
>>
>> This is an awesome question. File paths are so important to get
>> comfortable with as a web developer. Since the css files are in the css
>> folder, the file path needs to include a reference to the folder.
>>
>> The file path in the index.html file can be returned to what it was in
>> the starter code. Because the index.html file and the css folder are at the
>> same level, you can add css to the front of the file path to complete
>> the link. Have a look at the diff linked in the last message to see the
>> before and after.
>>
>> Because your advice file is in an advice folder, the path to your css
>> file is a little bit different. It needs to go up a level (to the root of
>> the project folder), then to the css folder to find your css file. When we
>> specify that we want to go up a level, we add ../ to the front of the
>> file path. Then we add the folder we have to go into to find the css file.
>> The path would look like this: "../css/denise-wilson-advice.css"
>>
>> Does that help? Reach out if there's anything I can clarify! Keep going -
>> you've got this!
>>
>> Best,
>> Lisa
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/skillcrush/git-practice/pull/1281#issuecomment-2143699799>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BGVBDB4RWDZ6DGY2ASMZYTLZFKSDNAVCNFSM6AAAAABIOA7XPWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCNBTGY4TSNZZHE>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

twosavoie commented:
Hi @Denisealisen,

Woo Hoo!! You did a great job! Well done sticking with it. That sort of grit will serve you well as a developer!

Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).

Best,
Lisa

Denisealisen commented:
Thank you so much!! That is awesome to know!

On Mon, Jun 3, 2024 at 10:28 AM Lisa Savoie ***@***.***>
wrote:

> Hi @Denisealisen <https://github.com/Denisealisen>,
>
> Woo Hoo!! You did a great job! Well done sticking with it. That sort of
> grit will serve you well as a developer!
>
> Your code is all merged in
> <https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif>
> .
>
> Best,
> Lisa
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/git-practice/pull/1281#issuecomment-2145511423>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGVBDB4OI4LAQUXKAMHI5JLZFSDQJAVCNFSM6AAAAABIOA7XPWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCNBVGUYTCNBSGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
